java/serving/src/main/java/feast/serving/config/RegistryConfig.java

```diff
@@ -59,7 +58,7 @@ RegistryFile registryFile(FeastProperties feastProperties, ApplicationContext co
         return new S3RegistryFile(context.getBean(AmazonS3.class), registryPath);
       case "":
       case "file":
-        return new LocalRegistryFile(Paths.get(registryPath));
+        return new LocalRegistryFile(registryPath);
```

Comment:
Can you explain this change?

Reply:
Trying to address this error in java unit tests:

```
Error:  /home/runner/work/feast/feast/java/serving/src/main/java/feast/serving/config/RegistryConfig.java:[62,47] incompatible types: java.nio.file.Path cannot be converted to java.lang.String
```

Reply:
Yeah, this is bug from my PR #2043, have no idea how it passed tests